tests/Admin/FieldDescriptionTest.php

```diff
@@ -22,6 +22,12 @@
 
 class FieldDescriptionTest extends TestCase
 {
+    /**
+     * NEXT_MAJOR: Remove the "legacy" group when `Sonata\DoctrineORMAdminBundle\Filter\EmptyFilter`
+     * class is removed or stops extending `NullFilter`.
+     *
+     * @group legacy
+     */
     public function testOptions(): void
```

Comment:
Why this test is related to EmptyFilter ?

Reply:
I don't know: https://github.com/sonata-project/SonataDoctrineORMAdminBundle/runs/2043015696?check_suite_focus=true#step:7:133.